tests/e2e/sample-applications/virt-todo/README.MD

```diff
@@ -0,0 +1,41 @@
+# notes:
+Obviously there are improvements to this workflow, however this is meant to
+serve as a template or example to fit into other working test frameworks.
+
+# To install
+setup the openshift-virtualization operator
+
+# Start vm
+`oc create -f namemspace-virttodo.yaml -f virtualmachine-fedora-close-carp.yaml -f service.yaml -f vmroute.yaml`
```

Comment:
create a single file, concatenated by `---`, to make command easier

Reply:
agree, however for a users readability and for creating docs from these files I would prefer to keep them separate.  The context of the reader is IMHO is a newb